pxemgr/ipxe_handlers.go

```diff
@@ -47,7 +47,7 @@ func (mgr *pxeManagerT) ipxeBootScript(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// for ignition we use only 1phase installation without mayu-infopusher
-	kernel := fmt.Sprintf("kernel %s/images/vmlinuz coreos.first_boot=1 initrd=initrd.cpio.gz coreos.config.url=%s?uuid=${uuid}&serial=${serial} systemd.journald.max_level_console=debug verbose log_buf_len=10M "+extraFlags+"\n", mgr.pxeURL(), mgr.ignitionURL())
+	kernel := fmt.Sprintf("kernel %s/images/vmlinuz flatcar.first_boot=1 initrd=initrd.cpio.gz coreos.config.url=%s?uuid=${uuid}&serial=${serial} systemd.journald.max_level_console=debug verbose log_buf_len=10M "+extraFlags+"\n", mgr.pxeURL(), mgr.ignitionURL())
```

Comment:
Need to check if `coreos.config.url` require change too

Reply:
seems that the change to support `coreos.config.url` also in Flatcar reached flatcar stable only later than 2247.6.0 see here https://github.com/flatcar-linux/ignition/pull/10

Seems than it is required to change it to `coreos.config.url`

Reply:
Changed this.